src/test/java/de/rabea/game/GameTypeTest.java

```diff
@@ -12,29 +14,83 @@
 
     FakeUserInterface fakeUserInterface;
     FakeComputerPlayer fakeComputerPlayer;
-    GameType gameType;
     GameManager gameManager;
+    Console fakeConsole;
 
     @Before
     public void setup() {
         fakeUserInterface = new FakeUserInterface();
         fakeComputerPlayer = new FakeComputerPlayer(O);
         gameManager = new GameManager(fakeUserInterface);
-        gameType = new GameType(fakeUserInterface, gameManager);
+        fakeConsole = new FakeConsole();
     }
 
     @Test
-    public void itSetsUpTheFirstGameAsAHvHGame() {
-        fakeUserInterface.provideConsoleInput("2");
-        Game game = gameType.createGame();
-        assertTrue(game != null);
+    public void createsAHumanVsHumanGame() {
+        GameType gameType = new GameType(new UIReturningHvH(fakeConsole), gameManager);
+        assertTrue(gameType.createGame() != null);
```

Comment:
I don't like tests that check x != null. Can you actually test some behaviour in these tests?